src/types/index.ts

```diff
@@ -34,7 +34,7 @@ export type LocalVideoContextType = {
 
 export type ContentShareControlContextType = {
   paused: boolean;
-  toggleContentShare: () => Promise<void>;
+  toggleContentShare: (sourceId?: string) => void;
```

Comment:
Changing the return type is a breaking change right? So can we keep the return type same? Also, in the usage example in the how-to, you are using `await toggleContentShare`.

Also, one query we pass an optional attribute to start with, so adding a required variable later would be a breaking change / hard later. I do not know how to handle this or this is fine. @ltrung @simmkyu can you advice here?

Reply:
Yeah, recommend keeping the return type.